Documentation/MenuObjects/Tmenu/Index.rst

```diff
@@ -517,16 +520,16 @@ showAccessRestrictedPages
     perform page-not-found handling when clicked (which can be used to
     capture the event and act accordingly of course). This means that the
     link's URL will point to the page even if it is not accessible by the
-    current frontend user. Note that the default behavior of page-not-found
+    current frontend user. Note that the default behavior of page not found
```

Comment:
Should be put in quotes to note a special term, else the dashes would be required so that "page-not-found" is all attributed to "handling". Without the dashes this association gets lost.